config-builder/src/main/java/com/ca/apim/gateway/cagatewayconfig/beans/Encass.java

```diff
@@ -156,4 +162,18 @@ public void sortArgumentsAndResults() {
         setArguments(getArguments().stream().collect(toCollection(() -> new TreeSet<>(Comparator.comparing(EncassArgument::getName)))));
         setResults(getResults().stream().collect(toCollection(() -> new TreeSet<>(Comparator.comparing(EncassResult::getName)))));
     }
+
+    @Override
+    public boolean hasBundleAnnotation(){
+        if(hasAnnotated()){
```

Comment:
This `hasAnnotated()` should be changed to `hasAnnotation()` or `isAnnotated()`.